k8s/prod-airqo-platform-frontend.yaml

```diff
@@ -23,6 +23,37 @@ spec:
           ports:
```

Comment:
I keep on wondering, is there a way we can explore the usage of one config file for each sub-repo and just change the specific ENVS for the different environments? More in the angle of using Helm charts? More: https://codefresh.io/helm-tutorial/helm-deployment-environments/

Reply:
Same case for the Jenkins files......

Reply:
@Baalmart, I have added helm chart capability